### PR TITLE
build: update to latest stable Chromium 83.0.4103

### PR DESCRIPTION
**Pull request by gregmagolan**
Also added in detailed instructions of the process to determine the URLs corresponding to Chromium version desired.

NB: updating to latest stable had issues in the past on a per platform basis

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by gregmagolan** [comment]
All green on Circle. @IgorMinar & @josephperrott , this may be a good one to try manually on OSX and Windows; I don't think Windows CI exercises any tests that use the rules_webtesting chromium

**Comment by gregmagolan** [comment]
The other spot that chrome is controlled is via the puppeteer version; that is easy to update in the root package.json + updating the map in `scripts/puppeteer-chrome-versions.js`:
```
// Mapping of puppeteer releases to their default Chrome version
// derived from https://github.com/puppeteer/puppeteer/blob/master/docs/api.md.
// The puppeteer package.json file contains the compatible Chrome revision such as
// "chromium_revision": "722234" but this does not map easily to the Chrome version
// so we use this mapping here instead.
module.exports = {
  "2.1.1": "80.0.3987.0",
  "2.1.0": "80.0.3987.0",
  "2.0.0": "79.0.3942.0",
  "1.20.0": "78.0.3882.0",
  "1.19.0": "77.0.3803.0",
  "1.17.0": "76.0.3803.0",
  "1.15.0": "75.0.3765.0",
  "1.13.0": "74.0.3723.0",
  "1.12.2": "73.0.3679.0",
};
```
Do you want me to align that version in this PR?

**Review by IgorMinar — CHANGES_REQUESTED** [review]
Yes. Please align the puppeteer version in this PR and document that process + cross reference it from the documentation you just wrote.

Thank you!

File: dev-infra/browsers/browser_repositories.bzl [hunk 1]
```diff
@@ -44,44 +44,76 @@ def platform_http_file(name, licenses, sha256, urls):
 def browser_repositories():
     """Load pinned rules_webtesting browser versions."""
 
+    # To update to a newer version of Chromium:
```
Comment (josephperrott) [inline dev-infra/browsers/browser_repositories.bzl:47]: Could we move this explanation to a README file?  The write up does a great job of explaining, but its tough to read inline vs markdown.

Comment (gregmagolan) [reply dev-infra/browsers/browser_repositories.bzl]: Done; put the instructions in dev-infra/browsers/README.md

**Comment by gregmagolan** [comment]
Also updates puppeteer controlled version to match

**Review by IgorMinar — CHANGES_REQUESTED** [review]
browsers/README.md missing, otherwise LGTM

File: dev-infra/browsers/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Chrome browser versions for testing and angular
```
Comment (IgorMinar) [inline dev-infra/browsers/README.md]:
```suggestion
# Chrome browser versions for testing and Angular
```

File: dev-infra/browsers/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Chrome browser versions for testing in angular
+
+The version of chrome used in tests is determined in a few places.
```
Comment (IgorMinar) [inline dev-infra/browsers/README.md:3]:
```suggestion
Within the Angular monorepo, we use Chrome to perform most of the local testing, and rely on Sauce Labs and BrowserStack to do cross-browser testing on our CI.

The version of Chrome used in tests within this monorepo is configured and controlled via `rules_webtesting` and `pupeteer`. We manually keep the configuration of these two tools in sync to create a consistent testing environment across unit, e2e, and integration tests.
```

File: dev-infra/browsers/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+# Chrome browser versions for testing in angular
```
Comment (IgorMinar) [inline dev-infra/browsers/README.md:1]:
```suggestion
# Browser configuration and versioning for testing of Angular
```

File: dev-infra/browsers/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+The version of chrome used in tests is determined in a few places.
+
+## rules_webtesting
+
+Bazel `karma_web_test_suit`e and `protractor_web_test_suite` targets will use Chromium provisioned by rules_webtesting. The version of chrome used is specified in the `dev-infra/browsers/browser_repositories.bzl` file. Its a bit of a process to upgrade the version in that file:
```
Comment (IgorMinar) [inline dev-infra/browsers/README.md:7]:
```suggestion
Bazel `karma_web_test_suite` and `protractor_web_test_suite` targets will use Chromium provisioned by rules_webtesting. The version of chrome used is specified in the `dev-infra/browsers/browser_repositories.bzl` file. The process of updating the Chrome version in that file consists of several steps:
```

**Review by IgorMinar — APPROVED** [review]
LGTM after the suggestions are resolved.

File: dev-infra/browsers/README.md (hunk 2 above)
Comment (gregmagolan) [reply dev-infra/browsers/README.md]: Done

File: dev-infra/browsers/README.md (hunk 3 above)
Comment (gregmagolan) [reply dev-infra/browsers/README.md]: 👍

File: dev-infra/browsers/README.md (hunk 4 above)
Comment (gregmagolan) [reply dev-infra/browsers/README.md]: 👍

File: dev-infra/browsers/README.md (hunk 5 above)
Comment (gregmagolan) [reply dev-infra/browsers/README.md]: 👍

**Comment by gregmagolan** [comment]
Tested locally on OSX: ```INFO: Elapsed time: 1816.460s, Critical Path: 569.21s
INFO: 2426 processes: 1865 darwin-sandbox, 18 local, 543 worker.
INFO: Build completed successfully, 30276 total actions
//dev-infra/commit-message:test                                          PASSED in 0.9s
//integration:cli-hello-world-ivy-compat_test                            PASSED in 258.6s
//integration:cli-hello-world-ivy-minimal_test                           PASSED in 243.2s
//integration:cli-hello-world_test                                       PASSED in 259.9s
//integration:hello_world__systemjs_umd_test                             PASSED in 65.9s
//integration:language_service_plugin_test                               PASSED in 33.5s
//integration:ng_update_test                                             PASSED in 31.3s
//integration:service-worker-schema_test                                 PASSED in 43.8s
//integration:terser_test                                                PASSED in 36.1s
//integration:typings_test_ts39_test                                     PASSED in 36.9s
//modules/benchmarks/src/change_detection/transplanted_views:e2e_chromium PASSED in 6.1s
//modules/benchmarks/src/largeform/ng2:e2e_chromium                      PASSED in 7.9s
//modules/benchmarks/src/largetable/baseline:e2e_chromium                PASSED in 6.8s
//modules/benchmarks/src/largetable/incremental_dom:e2e_chromium         PASSED in 5.8s
//modules/benchmarks/src/largetable/iv:e2e_chromium                      PASSED in 8.3s
//modules/benchmarks/src/largetable/ng2:e2e_chromium                     PASSED in 7.4s
//modules/benchmarks/src/largetable/ng2_switch:e2e_chromium              PASSED in 6.7s
//modules/benchmarks/src/largetable/render3:e2e_chromium                 PASSED in 13.1s
//modules/benchmarks/src/tree/baseline:e2e_chromium                      PASSED in 16.0s
//modules/benchmarks/src/tree/incremental_dom:e2e_chromium               PASSED in 7.6s
//modules/benchmarks/src/tree/iv:e2e_chromium                            PASSED in 10.0s
//modules/benchmarks/src/tree/ng1:e2e_chromium                           PASSED in 11.6s
//modules/benchmarks/src/tree/ng2:e2e_chromium                           PASSED in 11.9s
//modules/benchmarks/src/tree/ng2_next:e2e_chromium                      PASSED in 39.1s
//modules/benchmarks/src/tree/ng2_static:e2e_chromium                    PASSED in 10.2s
//modules/benchmarks/src/tree/ng2_switch:e2e_chromium                    PASSED in 8.8s
//modules/benchmarks/src/tree/render3:e2e_chromium                       PASSED in 9.8s
//modules/playground/e2e_test/async:protractor_tests_chromium            PASSED in 27.5s
//modules/playground/e2e_test/hello_world:protractor_tests_chromium      PASSED in 6.8s
//modules/playground/e2e_test/http:protractor_tests_chromium             PASSED in 9.4s
//modules/playground/e2e_test/jsonp:protractor_tests_chromium            PASSED in 5.5s
//modules/playground/e2e_test/key_events:protractor_tests_chromium       PASSED in 9.3s
//modules/playground/e2e_test/model_driven_forms:protractor_tests_chromium PASSED in 6.2s
//modules/playground/e2e_test/order_management:protractor_tests_chromium PASSED in 4.7s
//modules/playground/e2e_test/person_management:protractor_tests_chromium PASSED in 6.1s
//modules/playground/e2e_test/relative_assets:protractor_tests_chromium  PASSED in 5.6s
//modules/playground/e2e_test/routing:protractor_tests_chromium          PASSED in 8.1s
//modules/playground/e2e_test/sourcemap:protractor_tests_chromium        PASSED in 3.9s
//modules/playground/e2e_test/svg:protractor_tests_chromium              PASSED in 14.6s
//modules/playground/e2e_test/template_driven_forms:protractor_tests_chromium PASSED in 5.9s
//modules/playground/e2e_test/upgrade:protractor_tests_chromium          PASSED in 6.8s
//modules/playground/e2e_test/web_workers/animations:protractor_tests_chromium PASSED in 3.5s
//modules/playground/e2e_test/web_workers/input:protractor_tests_chromium PASSED in 15.0s
//modules/playground/e2e_test/web_workers/kitchen_sink:protractor_tests_chromium PASSED in 50.2s
//modules/playground/e2e_test/web_workers/message_broker:protractor_tests_chromium PASSED in 12.9s
//modules/playground/e2e_test/web_workers/router:protractor_tests_chromium PASSED in 11.3s
//modules/playground/e2e_test/web_workers/todo:protractor_tests_chromium PASSED in 10.9s
//modules/playground/e2e_test/zippy_component:protractor_tests_chromium  PASSED in 12.6s
//packages/animations/browser/test:circular_deps_test                    PASSED in 7.1s
//packages/animations/browser/test:test                                  PASSED in 2.2s
//packages/animations/browser/test:test_web_chromium                     PASSED in 6.0s
//packages/animations/browser/test:testing_circular_deps_test            PASSED in 7.0s
//packages/animations/test:circular_deps_test                            PASSED in 2.8s
//packages/animations/test:test                                          PASSED in 2.5s
//packages/animations/test:test_web_chromium                             PASSED in 5.3s
//packages/bazel/test/ngc-wrapped:flat_module_test                       PASSED in 1.3s
//packages/bazel/test/ngc-wrapped:ngc_test                               PASSED in 6.2s
//packages/benchpress/test:test                                          PASSED in 3.3s
//packages/common/http/test:circular_deps_test                           PASSED in 6.7s
//packages/common/http/test:test                                         PASSED in 5.1s
//packages/common/http/test:test_web_chromium                            PASSED in 6.7s
//packages/common/http/testing/test:circular_deps_test                   PASSED in 5.8s
//packages/common/http/testing/test:test                                 PASSED in 5.4s
//packages/common/http/testing/test:test_web_chromium                    PASSED in 7.7s
//packages/common/test:circular_deps_test                                PASSED in 17.3s
//packages/common/test:test                                              PASSED in 6.8s
//packages/common/test:test_web_chromium                                 PASSED in 10.5s
//packages/common/test:testing_circular_deps_test                        PASSED in 7.3s
//packages/common/upgrade/test:circular_deps_test                        PASSED in 9.1s
//packages/common/upgrade/test:test                                      PASSED in 2.9s
//packages/compiler-cli/integrationtest/bazel/injectable_def/app/test:test PASSED in 6.3s
//packages/compiler-cli/integrationtest/bazel/injector_def/ivy_build/app/test:test PASSED in 4.2s
//packages/compiler-cli/ngcc/test:test                                   PASSED in 39.2s
//packages/compiler-cli/src/ngtsc/annotations/test:test                  PASSED in 3.9s
//packages/compiler-cli/src/ngtsc/core/test:test                         PASSED in 2.1s
//packages/compiler-cli/src/ngtsc/cycles/test:test                       PASSED in 2.1s
//packages/compiler-cli/src/ngtsc/entry_point/test:test                  PASSED in 0.9s
//packages/compiler-cli/src/ngtsc/file_system/test:test                  PASSED in 1.1s
//packages/compiler-cli/src/ngtsc/imports/test:test                      PASSED in 2.8s
//packages/compiler-cli/src/ngtsc/indexer/test:test                      PASSED in 1.9s
//packages/compiler-cli/src/ngtsc/partial_evaluator/test:test            PASSED in 4.9s
//packages/compiler-cli/src/ngtsc/reflection/test:test                   PASSED in 1.2s
//packages/compiler-cli/src/ngtsc/scope/test:test                        PASSED in 2.1s
//packages/compiler-cli/src/ngtsc/shims/test:test                        PASSED in 1.6s
//packages/compiler-cli/src/ngtsc/transform/test:test                    PASSED in 2.1s
//packages/compiler-cli/src/ngtsc/typecheck/test:test                    PASSED in 12.3s
//packages/compiler-cli/src/ngtsc/util/test:test                         PASSED in 4.1s
//packages/compiler-cli/test:perform_compile                             PASSED in 2.8s
//packages/compiler-cli/test:perform_watch                               PASSED in 22.8s
//packages/compiler-cli/test/diagnostics:typescript_version              PASSED in 4.8s
//packages/compiler-cli/test/metadata:test                               PASSED in 19.3s
//packages/compiler/test:circular_deps_test                              PASSED in 7.4s
//packages/compiler/test/css_parser:css_parser                           PASSED in 2.2s
//packages/compiler/test/css_parser:css_parser_web_chromium              PASSED in 5.7s
//packages/compiler/test/expression_parser:expression_parser             PASSED in 2.4s
//packages/compiler/test/expression_parser:expression_parser_web_chromium PASSED in 8.7s
//packages/compiler/test/ml_parser:ml_parser                             PASSED in 2.8s
//packages/compiler/test/ml_parser:ml_parser_web_chromium                PASSED in 7.6s
//packages/compiler/test/render3:test                                    PASSED in 3.3s
//packages/compiler/test/selector:selector                               PASSED in 2.0s
//packages/compiler/test/selector:selector_web_chromium                  PASSED in 5.3s
//packages/core/schematics/test:test                                     PASSED in 23.6s
//packages/core/schematics/test/google3:google3                          PASSED in 44.5s
//packages/core/test:circular_deps_test                                  PASSED in 7.1s
//packages/core/test:test_web_chromium                                   PASSED in 24.4s
//packages/core/test:testing_circular_deps_test                          PASSED in 10.4s
//packages/core/test/acceptance:acceptance                               PASSED in 12.2s
//packages/core/test/acceptance:acceptance_web_chromium                  PASSED in 53.8s
//packages/core/test/bundling/cyclic_import:symbol_test                  PASSED in 1.9s
//packages/core/test/bundling/cyclic_import:test                         PASSED in 2.0s
//packages/core/test/bundling/hello_world:symbol_test                    PASSED in 0.7s
//packages/core/test/bundling/hello_world:test                           PASSED in 4.6s
//packages/core/test/bundling/hello_world_r2:test                        PASSED in 0.9s
//packages/core/test/bundling/injection:symbol_test                      PASSED in 0.7s
//packages/core/test/bundling/injection:test                             PASSED in 2.2s
//packages/core/test/bundling/todo:symbol_test                           PASSED in 0.9s
//packages/core/test/bundling/todo:test                                  PASSED in 1.6s
//packages/core/test/bundling/todo_i18n:test                             PASSED in 2.8s
//packages/core/test/bundling/todo_r2:test                               PASSED in 1.8s
//packages/core/test/render3:render3                                     PASSED in 3.2s
//packages/core/test/render3:render3_web_chromium                        PASSED in 5.9s
//packages/core/test/render3/ivy:ivy                                     PASSED in 2.0s
//packages/core/test/render3/perf:perf                                   PASSED in 0.6s
//packages/core/test/strict_types:strict_types                           PASSED in 0.5s
//packages/elements/schematics/ng-add:test                               PASSED in 2.0s
//packages/elements/test:circular_deps_test                              PASSED in 5.3s
//packages/elements/test:test_chromium                                   PASSED in 6.4s
//packages/examples/common:protractor_tests_chromium                     PASSED in 25.4s
//packages/examples/core:protractor_tests_chromium                       PASSED in 10.7s
//packages/examples/core:test                                            PASSED in 3.3s
//packages/examples/core/testing/ts:test                                 PASSED in 4.1s
//packages/examples/forms:protractor_tests_chromium                      PASSED in 36.5s
//packages/examples/service-worker/push:protractor_tests_chromium        PASSED in 3.8s
//packages/examples/service-worker/registration-options:protractor_tests_chromium PASSED in 6.3s
//packages/examples/upgrade/static/ts/full:full_protractor_chromium      PASSED in 7.0s
//packages/examples/upgrade/static/ts/lite:lite_protractor_chromium      PASSED in 8.8s
//packages/examples/upgrade/static/ts/lite-multi:lite-multi_protractor_chromium PASSED in 7.4s
//packages/examples/upgrade/static/ts/lite-multi-shared:lite-multi-shared_protractor_chromium PASSED in 5.6s
//packages/forms/test:circular_deps_test                                 PASSED in 6.6s
//packages/forms/test:test                                               PASSED in 12.6s
//packages/forms/test:test_web_chromium                                  PASSED in 14.7s
//packages/language-service/test:circular_deps_test                      PASSED in 9.0s
//packages/localize/schematics/ng-add:test                               PASSED in 3.8s
//packages/localize/src/localize/test:test                               PASSED in 0.5s
//packages/localize/src/tools/test:test                                  PASSED in 7.2s
//packages/localize/src/tools/test/translate/integration:integration     PASSED in 2.7s
//packages/localize/src/utils/test:test                                  PASSED in 1.1s
//packages/localize/test:circular_deps_test                              PASSED in 5.5s
//packages/localize/test:test                                            PASSED in 1.0s
//packages/platform-browser-dynamic/test:circular_deps_test              PASSED in 9.0s
//packages/platform-browser-dynamic/test:test                            PASSED in 1.8s
//packages/platform-browser-dynamic/test:test_web_chromium               PASSED in 5.2s
//packages/platform-browser/animations/test:circular_deps_test           PASSED in 6.1s
//packages/platform-browser/animations/test:test                         PASSED in 2.4s
//packages/platform-browser/animations/test:test_web_chromium            PASSED in 12.4s
//packages/platform-browser/test:circular_deps_test                      PASSED in 7.3s
//packages/platform-browser/test:test                                    PASSED in 2.7s
//packages/platform-browser/test:test_web_chromium                       PASSED in 24.8s
//packages/platform-browser/test:testing_circular_deps_test              PASSED in 6.8s
//packages/platform-server/test:circular_deps_test                       PASSED in 12.8s
//packages/platform-server/test:test                                     PASSED in 2.0s
//packages/platform-server/test:testing_circular_deps_test               PASSED in 37.9s
//packages/platform-webworker/test:test                                  PASSED in 2.6s
//packages/platform-webworker/test:test_web_chromium                     PASSED in 6.5s
//packages/router/test:circular_deps_test                                PASSED in 6.1s
//packages/router/test:test                                              PASSED in 6.8s
//packages/router/test:test_web_chromium                                 PASSED in 14.0s
//packages/router/test:testing_circular_deps_test                        PASSED in 6.6s
//packages/router/test/aot_ngsummary_test:test                           PASSED in 2.5s
//packages/router/upgrade/test:circular_deps_test                        PASSED in 18.7s
//packages/router/upgrade/test:test_web_chromium                         PASSED in 5.3s
//packages/service-worker/config/test:circular_deps_test                 PASSED in 1.1s
//packages/service-worker/config/test:test                               PASSED in 4.2s
//packages/service-worker/test:circular_deps_test                        PASSED in 15.4s
//packages/service-worker/test:test                                      PASSED in 2.9s
//packages/service-worker/test:test_web_chromium                         PASSED in 5.4s
//packages/service-worker/worker/test:circular_deps_test                 PASSED in 2.0s
//packages/service-worker/worker/test:test                               PASSED in 1.7s
//packages/upgrade/src/common/test:circular_deps_test                    PASSED in 13.2s
//packages/upgrade/src/common/test:test_chromium                         PASSED in 7.2s
//packages/upgrade/src/dynamic/test:test_chromium                        PASSED in 17.0s
//packages/upgrade/static/test:circular_deps_test                        PASSED in 5.7s
//packages/upgrade/static/test:test_chromium                             PASSED in 11.8s
//packages/upgrade/static/testing/test:circular_deps_test                PASSED in 8.0s
//packages/upgrade/static/testing/test:test_chromium                     PASSED in 6.5s
//packages/zone.js/test:browser_disable_wrap_uncaught_promise_rejection_karma_jasmine_test_chromium PASSED in 2.7s
//packages/zone.js/test:browser_green_test_karma_jasmine_test_chromium   PASSED in 10.0s
//packages/zone.js/test:browser_legacy_test_karma_jasmine_test_chromium  PASSED in 13.3s
//packages/zone.js/test:browser_shadydom_karma_jasmine_test_chromium     PASSED in 3.8s
//packages/zone.js/test:browser_test_karma_jasmine_test_chromium         PASSED in 9.1s
//packages/zone.js/test:karma_jasmine_test_ci_chromium                   PASSED in 2.0s
//packages/zone.js/test:test_node                                        PASSED in 7.1s
//packages/zone.js/test:test_node_bluebird                               PASSED in 1.5s
//packages/zone.js/test:test_node_error_disable_policy                   PASSED in 1.8s
//packages/zone.js/test:test_node_error_lazy_policy                      PASSED in 1.4s
//packages/zone.js/test:test_node_no_jasmine_clock                       PASSED in 7.3s
//packages/zone.js/test:test_npm_package                                 PASSED in 0.6s
//tools/size-tracking:test                                               PASSED in 0.5s
//tools/symbol-extractor:test                                            PASSED in 1.0s
//tools/testing:fail_bootstrap_test                                      PASSED in 2.5s
//tools/testing:fail_test                                                PASSED in 0.7s
//tools/ts-api-guardian:tests                                            PASSED in 24.8s
//packages/compiler-cli/test/compliance:compliance                       PASSED in 90.8s
  Stats over 4 runs: max = 90.8s, min = 53.2s, avg = 67.2s, dev = 14.6s
//packages/compiler-cli/test/ngtsc:ngtsc                                 PASSED in 311.8s
  Stats over 4 runs: max = 311.8s, min = 133.4s, avg = 180.5s, dev = 75.9s
//packages/core/test:test                                                PASSED in 7.8s
  Stats over 4 runs: max = 7.8s, min = 7.2s, avg = 7.5s, dev = 0.3s

Executed 205 out of 205 tests: 205 tests pass.
INFO: Build completed successfully, 30276 total actions
✨  Done in 1818.10s.
```

**Pull request closed without merging**